src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/UtcNodeFactory.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using ILCompiler.DependencyAnalysis;
+using ILCompiler.DependencyAnalysisFramework;
+using Internal.Runtime;
+using Internal.TypeSystem;
+using Internal.TypeSystem.Ecma;
+
+namespace ILCompiler
+{
+    public class UtcNodeFactory : NodeFactory
+    {
+        public static string CompilationUnitPrefix = "";
```

Comment:
This should be on the `NameMangler.CompilationUnitPrefix` plan. I'll have to go fix it internally so I can test the change before I change it here.